modules/modules.go

```diff
@@ -0,0 +1,61 @@
+package modules
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/hooks"
+)
+
+func NewBuilder() Builder {
+	return &builder{builders()}
+}
+
+type Builder interface {
+	SetModuleBuilders(builders ModuleBuilders) Builder
```

Comment:
Let's omit the `SetModuleBuilders` since it's only used for tests. The tests are the same package so it can directly set the builders, such as:

```
builder := &builder{
  builders: ModuleBuilders{
    vendor: {
      moduleName: func(cfg json.RawMessage, client *http.Client) (interface{}, error) {
      return test.givenModule, test.givenHookBuilderErr
    },
  },
}
```

Reply:
Great advice, thanks!